tests/Microsoft.Bot.Builder.TestBot/Startup.cs

```diff
@@ -97,18 +43,16 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             {
                 app.UseDeveloperExceptionPage();
             }
+            else
+            {
+                app.UseHsts();
```

Comment:
I'm a bit skeptical on this - it's an awesome feature, but I'm not at all clear on the implications. Do we need to gen/maintain a X.509 Cert?

Reply:
this is basically the default template - I was deliberately doing as few edits as possible - maybe we should discuss what you want to do here